src/components/dialogs/create-filter-dialog.js

```diff
@@ -209,8 +211,9 @@ const CreateFilterDialog = ({
     }, [tableValues]);
 
     const handleValidation = () => {
-        //To manage the case when we never tried to enter a name
+        setValidationsCount((prev) => prev + 1);
```

Comment:
I dont understand the purpose of this counter ? maybe we can discuss about it, dont reply here.

Reply:
It carries the information that validation number has changed or not, allowing to not mark as a problem something like an empty property line after it has been added but has it marked after a new validation.